src/main/java/competition/subsystems/lights/LightsCommunicationSubsystem.java

```diff
@@ -120,7 +122,10 @@ public void periodic() {
             }
         } else if (collector.getGamePieceCollected()) {
             currentState = LightsStateMessage.GamePieceCollected;
-        } else if (dsEnabled) {
+        } else if(arm.isMaintainerAtGoal() && arm.upperArm.getArmPositionInDegrees() > 20){
```

Comment:
I think we wanted to check the x coordinate of the arm, not the angle, but maybe this still covers all of the scoring positions?

Reply:
Yeah, this seems like it would probably work. The upper arm tends to be at ~45 degrees or greater for most interesting motions.